core/templates/dev/head/pages/exploration_player/PredictionAlgorithmSampleService.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Prediction Algorithm sample for testing framework.
+ */
+
+ oppia.factory('PredictionAlgorithmSampleService', [function() {
```

Comment:
Nope. We should have a "simple" sample service for testing in any case, regardless of whatever is being used in production, because the production services are going to be quite complex and not suitable for debugging pipeline issues that arise in testing. You can probably provide a custom service in tests using $provide.factory().